itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/parse/TestReplicationScenariosExternalTables.java

```diff
@@ -904,8 +901,20 @@ public void replicationWithTableNameContainsKeywords() throws Throwable {
     return ReplicationTestUtils.externalTableBasePathWithClause(REPLICA_EXTERNAL_BASE, replica);
   }
 
-  private void assertExternalFileInfo(List<String> expected, Path externalTableInfoFile)
+  private void assertExternalFileInfo(List<String> expected, String dumplocation) throws IOException {
+    assertExternalFileInfo(expected, dumplocation, null);
+  }
+  private void assertExternalFileInfo(List<String> expected, String dumplocation, String dbName)
       throws IOException {
+    Path externalTableInfoFile = new Path(dumplocation, relativeExtInfoPath(dbName));
     ReplicationTestUtils.assertExternalFileInfo(primary, expected, externalTableInfoFile);
   }
+  private String relativeExtInfoPath(String dbName) {
+
+    if (dbName == null) {
```

Comment:
For incremental dbname is not needed?

Reply:
No, the location of external table info file is different in bootstrap and incremental case.